apps/prairielearn/src/server.js

```diff
@@ -1842,6 +1862,23 @@ export async function initExpress() {
     },
     (await import('./pages/instructorQuestions/instructorQuestions.js')).default,
   ]);
+  app.use('/pl/course/:course_id(\\d+)/ai_generate_question_audit', [
+    function (req, res, next) {
+      res.locals.navSubPage = 'questions';
+      next();
+    },
+    (await import('./ee/pages/instructorAiGenerateJobReview/instructorAiGenerateJobReview.js'))
+      .default,
+  ]);
+  app.use('/pl/course/:course_id(\\d+)/ai_generate_question_results/:job_id(\\d+)', [
+    function (req, res, next) {
+      res.locals.navSubPage = 'questions';
+      res.locals.lookupParams = req.params;
```

Comment:
Are you using this because you couldn't access these params inside the route handler? If so, you should use `mergeParams: true` when creating the router object. See the docs here: https://expressjs.com/en/api.html. You can find examples of this throughout the PL codebase.

Reply:
changed in 3c8bafd1470728a8c79b488213de1dfae0aed79e.